app/webpacker/styles/_govuk_overrides.scss

```diff
@@ -5,3 +5,9 @@
 .no-wrap {
   white-space: nowrap;
 }
+
+.govuk-select .filter-select__start-year {
```

Comment:
it doesn't look like this is being rendered?

Reply:
I think the filter isn't showing because that provider only has one year of trainees. The CSS issue is likely to do with my shoddy CSS skills. I'll have a look at that. And the no filter year thing.

Reply:
> I think the filter isn't showing because that provider only has one year of trainees

Of course, sensible 👍🏻 

> The trainee in the future hasn't had a year created because we don't have an AcademicCycle in the system for that far ahead. The latest one is for 22/23.

Can we make the validation or the academic cycle consistent? Either the user can't create a trainee for that year. Or they can and it'll render a year in the start year select.